MapboxSearch/offline/src/main/java/com/mapbox/search/offline/OfflineSearchCallbackAdapter.kt

```diff
@@ -7,8 +7,10 @@ import com.mapbox.search.base.result.BaseSearchResult
 internal class OfflineSearchCallbackAdapter(private val callback: OfflineSearchCallback) : BaseSearchCallback {
     override fun onResults(results: List<BaseSearchResult>, responseInfo: BaseResponseInfo) {
         callback.onResults(
-            results.map { OfflineSearchResult(it.rawSearchResult) },
-            OfflineResponseInfo(responseInfo.requestOptions.core.mapToOfflineSdkType())
+            results = results
+                .map { OfflineSearchResult(it.rawSearchResult) }
```

Comment:
nit 

As an optimisation, it's better to filter before map in order to avoid unnecessary allocations

Reply:
Thank you for the advice, I'll keep it in mind for the future. Here it's not a concern since we just return the results to the UI, it doesn't have any performance implications.

Reply:
Although, I did update as you suggested :)